### PR TITLE
Sr/issue 70

### PR DESCRIPTION
**Pull request by sofiarm21**
Update About Us section

### CODE REVIEW

**Comment by sofiarm21** [comment]
Dave Corkle profile photo missing

File: src/content/english.js [hunk 1]
```diff
@@ -218,44 +218,50 @@ const TESTIMONIALS = [
         name: 'Oscar Lafarga',
         quote: 'SetLife has become my long-term commitment to a sustainable future when it comes to technology and its impact on society. In a world where information & communication moves at the speed of light, constraining our education to the classroom leaves us woefully under-equipped to deal with the major global problems we face. My goal is to incentivize teaching as one of the highest-paid and most valued professions, toppling the knowledge-hoarding power structures that sustain a global inequality of wealth',
         date: 'President & CTO',
-        portrait: 'https://user-images.githubusercontent.com/4914611/63651576-6e1cb200-c756-11e9-88f2-87cb80faa5a1.png'
+        portrait: 'https://user-images.githubusercontent.com/49292858/68433886-0808ce00-018e-11ea-9818-64cd4372bdc6.png'
+    },
+    {
```
Comment (otech47) [inline src/content/english.js]: Let's sort these by putting those with testimonials first so Oscar, Viki, Quinn, Sofia, followed by the rest

File: src/content/english.js [hunk 2]
```diff
@@ -218,44 +218,50 @@ const TESTIMONIALS = [
...
+        portrait: 'https://user-images.githubusercontent.com/49292858/68434040-5fa73980-018e-11ea-9ab5-079506b65ad2.jpg'
     },
     {
         name: 'Quinn Pruitt',
         quote: 'Project based learning is by far the most effective learning environment for software development. In seven weeks, I learned more about backend development and product management than I would have learned in an entire semester in college. The projects were quite challenging, yet incredibly rewarding. The high level of autonomy and personal responsibility provided a strong sense of ownership for my work, which I found crucial to my growth and success. I would recommend this program to anyone looking to level up their professional skills in a fun an engaging learning environment.',
         date: 'Technical Advisor',
-        portrait: 'https://media.licdn.com/dms/image/C5603AQEtPbVdzw90IQ/profile-displayphoto-shrink_800_800/0?e=1572480000&v=beta&t=FQ4lOzdfOXfou0PvkotYPpP59hdqDXFEJsntss3b7q4'
+        portrait: 'https://user-images.githubusercontent.com/49292858/68435657-39839880-0192-11ea-95ab-43c9f1bd8c91.jpg'
```
Comment (otech47) [inline src/content/english.js]: Use this image instead: https://user-images.githubusercontent.com/4914611/68456387-e7785c80-01fd-11ea-91ee-14dec845c871.jpeg

**Comment by sofiarm21** [comment]
Ready for merge

**Pull request merged**